core/corehttp/gateway_handler.go

```diff
@@ -83,6 +83,7 @@ func (sw *statusResponseWriter) WriteHeader(code int) {
 	if redirect != "" && code == http.StatusOK {
 		code = http.StatusMovedPermanently
 	}
+	log.Debugw("subdomain redirect", "location", redirect, "status", code)
```

Comment:
This will always log a redirect, even if we didn't redirect.

Reply:
🤦 fixed, thanks!